search/search_quality/assessment_tool/edits.hpp

```diff
@@ -17,11 +17,22 @@ class Edits
 
   struct Entry
   {
+    enum class Origin
+    {
+      Loaded,
+      Created
+    };
+
     Entry() = default;
+    Entry(Relevance relevance, Origin origin)
+      : m_curr(relevance), m_orig(relevance), m_origin(origin)
```

Comment:
Let's rename Origin to Type. parentage, ancestry and family are not so relevant to the meaning of the field.